lib/galaxy/model/__init__.py

```diff
@@ -2457,6 +2458,46 @@ def __init__(self, user, group):
         self.group = group
 
 
+class Notification(Base, Dictifiable, RepresentById):
+    __tablename__ = "notification"
+
+    id = Column(Integer, primary_key=True)
+    create_time = Column(DateTime, default=now)
+    update_time = Column(DateTime, default=now, onupdate=now)
+    publication_time = Column(DateTime, default=now)
+    expiration_time = Column(DateTime, default=now() + timedelta(days=30 * 6))
+    source = Column(String(32), index=True)
+    category = Column(String(64), index=True)
+    variant = Column(String(16), index=True)
```

Comment:
maybe add very brief inline comments (on the right?) explaining what there are? Like what you have in the data models table in the description of this PR. That would be very helpful for anyone reading the model definition code down the road.